mail_service/worker_notify_user.py

```diff
@@ -100,8 +100,8 @@ def __init__(self, job_info):
             logger.debug("Processing mail for failed build.")
             self.image_under_test = job_info.get("project_name")
 
-        self.project = self.job_info["namespace"] + "/" + \
-            self.job_info["job_name"]
+        self.project = self.job_info["namespace"].split("-")[0] + \
+            "/" + self.job_info["job_name"]
 
```

Comment:
@navidshaikh may be we remove the job-name here and keep the name space as it is. this will make sure to keep uniqueness even for the image with different tag